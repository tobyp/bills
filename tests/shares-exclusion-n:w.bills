#!./bills
A	ABC\2AB.	2.00
