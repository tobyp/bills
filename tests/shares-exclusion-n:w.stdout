todo: C pays 2.0 to A
